owncloudApp/src/main/res/values/strings.xml

```diff
@@ -758,5 +758,7 @@
     <string name="ocis_accounts_warning_message">Please, remove the account and login again to get the spaces feature</string>
     <string name="ocis_accounts_warning_checkbox_message">Don\'t show again</string>
     <string name="ocis_accounts_warning_button">Understood</string>
+    <string name="tab_label_markdown">Markdown</string>
+    <string name="tab_label_ascii">Text</string>
```

Comment:
Ah! Indeed you added them here! It's just that you didn't use them 😄. Use them in the code and move the "Markdown" string to the non-translatable ones 😁